tree_sitter_v/package.json

```diff
@@ -16,8 +16,8 @@
 		"install": "node-gyp-build",
 		"prebuildify": "prebuildify --napi --strip",
 		"lint": "eslint \"**/*.js\"",
-		"format": "prettier --write \"**/*.js\"",
-		"format:check": "prettier --check \"**/*.js\""
+		"format": "prettier --write \"grammar.js\"",
+		"format:check": "prettier --check \"grammar.js\""
```

Comment:
We have multiple js files in that directory. Prettier can take care of all of them.